backend/experiment/tests/test_frontend_style.py

```diff
@@ -0,0 +1,64 @@
+import unittest
+
+from experiment.actions.frontend_style import FrontendStyle
+
+class TestFrontendStyle(unittest.TestCase):
+
+    def test_init_with_valid_root_style(self):
+        style = FrontendStyle(FrontendStyle.PRIMARY)
```

Comment:
The initialization of `FrontendStyle` is somewhat awkward, I think - it now looks as if it's both a class and an enum.

Reply:
Yeah, I was struggling to find the best way to do it :-( Would you have any suggestions? Does python have enums?

Reply:
Yes, it does! [https://docs.python.org/3/library/enum.html](https://docs.python.org/3/library/enum.html)

Reply:
I think it would be neater to have a separate Enum with all the style options, and the class which is initialized with one of these options.